Makefile

```diff
@@ -63,11 +63,13 @@ notes: check-bump
 	towncrier build --yes --version $(UPCOMING_VERSION)
 	# Before we bump the version, make sure that the towncrier-generated docs will build
 	make build-docs
-	git commit -m "Compile release notes"
+	git commit -m "Compile release notes for v$(UPCOMING_VERSION)"
 
 release: check-bump clean
 	# require that upstream is configured for ethereum/<REPO_NAME>
-	git remote -v | grep "upstream\tgit@github.com:ethereum/<REPO_NAME>.git (push)\|upstream\thttps://github.com/ethereum/<REPO_NAME> (push)"
+	@git remote -v | grep \
+		-e "upstream\tgit@github.com:ethereum/<REPO_NAME>.git (push)" \
+		-Ee "upstream\thttps://(www.)?github.com/ethereum/<REPO_NAME> \(push\)"
```

Comment:
I like `-e` as a flag indicating that line is a regex. 

TIL: `-E` is an 'extended regex' but could you explain what it's purpose is on this line?

Reply:
- `-e` here more importantly allows us to use separate strings / lines so it makes the whole command easier to read. That's why the first command did not change from what it used to be.

- `-E` in the second command gives back meaning to some special characters, like the `(` and `)`, which allowed me to add `(www.)?` for whether or not `www.` exists. In doing so, we have to backslash the last two parentheses in `(push)` since they now hold special meaning. The `-e` is necessary here as well to designate a new string / pattern and allows us to put it in a separate line for easy reading.